### DISCUSSION TITLE
Is it possible to import the catalogs, databases, and tables from the existing cluster into Amoro for management?

### DISCUSSION THREAD

**Discussion by HwiLu** [apache/amoro#3063] [Q&A] [answered]
I have a data lake cluster using Iceberg and Paimon. Is it possible to manage the existing tables in the cluster using Amoro? It seems that the official website doesn't provide related documentation.

**Comment by zhoujinsong** [comment, marked as answer]
Yes.

What you need to do is create an external catalog to help Amoro explore your existing tables.
You can find related information here: https://amoro.apache.org/docs/latest/managing-catalogs/

**Reply by zhoujinsong** [reply]
BTW, if you are using version 0.6.1 or an earlier version, you may need to create separate catalogs for Iceberg and Paimon tables.
If you are using the 0.7.0-SNAPSHOT (master branch), you can try to create one catalog for both Iceberg and Paimon tables.

**Reply by HwiLu** [reply]
I have a question: @zhoujinsong 
Will the new catalog I create affect the existing catalogs? For example, if I create a catalog named `catalog_ex` and I already have a catalog named `catalog_spark`, can the new `catalog_ex` read the databases under `catalog_spark`? If it can, will optimizing `catalog_ex` affect applications that use the previous catalog, `catalog_spark`, to read data from the data lake?

I don’t quite understand how catalogs work. I thought they were just directories. Why can a newly created catalog read databases under an old catalog?

If you're willing to explain, I would greatly appreciate it.
